src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void mark() {
+        this.isDone = true;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void unMark() {
```

Comment:
Nice attempt at using camel case, but I believe 'unmark' should be treated as a single term